common/lib/xmodule/xmodule/js/src/video/04_video_full_screen.js

```diff
@@ -11,82 +11,134 @@
             '</button>'
         ].join('');
 
-    // VideoControl() function - what this module "exports".
-        return function(state) {
-            var dfd = $.Deferred();
+    // The following properties and functions enable cross-browser use of the
+    // the Fullscreen Web API.
+    //
+    //     function getVendorPrefixed(property)
+    //     function getFullscreenElement()
+    //     function exitFullscreen()
+    //     function requestFullscreen(element, options)
+    //
+    //     For more information about the Fullscreen Web API see MDN:
+    //     https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API
+    //     way - you don't have to do repeated jQuery element selects.
```

Comment:
This line of the comment feels like leftover from some other comment, perhaps?

Reply:
Good catch